sdk/monitor/monitor-opentelemetry/review/monitor-opentelemetry.api.md

```diff
@@ -15,8 +17,10 @@ export interface AzureMonitorOpenTelemetryOptions {
     enableLiveMetrics?: boolean;
     enableStandardMetrics?: boolean;
     instrumentationOptions?: InstrumentationOptions;
+    logRecordProcessors?: [LogRecordProcessor];
     resource?: Resource;
     samplingRatio?: number;
+    spanProcessors?: [SpanProcessor];
```

Comment:
Why no metric readers?

Reply:
Is there any problem with adding the metric reader after we initialize distro?, we are adding these configs as processors order matters and batch processors should be added at the end. Customer should be able to add metric reader after at any point

Reply:
Don't customers need to be able to add metric readers before ours in the same way they need it for spans and logs?